package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "workshop-protractor",
+  "version": "1.0.0",
+  "description": "This is a Workshop about Protractor",
+  "main": "index.js",
+  "scripts": {
+    "clean": "if exist dist rd /s /q dist",
```

Comment:
use Linux commands.

Also, change the console, use git bash, cmder, Embed Linux